presto-docs/src/main/sphinx/sql/select.rst

```diff
@@ -1061,7 +1061,13 @@ standard rules for nulls. The subquery must produce exactly one column::
 
     SELECT name
     FROM nation
-    WHERE regionkey IN (SELECT regionkey FROM region)
+    WHERE regionkey IN (SELECT regionkey FROM region WHERE name = 'AMERICA' OR name = 'AFRICA')
+ 
+The ``IN`` predicate can also test against a static list of values::
+ 
+    SELECT name
+    FROM nation
+    WHERE regionkey IN (1, 3)
```

Comment:
Can you add semicolon to this and the other two examples above, so that they match the others?